compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1707,6 +1715,18 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
         }
     }
 
+    fn encode_debugger_visualizers(&mut self) -> Lazy<[DebuggerVisualizerFile]> {
+        empty_proc_macro!(self);
+
+        let debugger_visualizers = self
+            .tcx
+            .debugger_visualizers(())
+            .iter()
+            .map(|visualizer| Lrc::new(visualizer.clone()))
+            .collect::<Vec<_>>();
+        self.lazy(debugger_visualizers.iter().map(|rc| &**rc))
```

Comment:
You can replace this by `self.lazy(self.tcx.debugger_visualizers(()).iter())`. No need to clone the visualizers.